mods/ra/rules/structures.yaml

```diff
@@ -568,7 +568,7 @@ PBOX:
 		Armaments: garrisoned
 		PortOffsets: 384,0,128, 224,-341,128, -224,-341,128, -384,0,128, -224,341,128, 224,341,128
 		PortYaws: 0, 176, 341, 512, 682, 853
-		PortCones: 86, 86, 86, 86, 86, 86
+		PortCones: 100, 100, 100, 100, 100, 100
```

Comment:
IIRC from my own earlier tests increasing these to 87 should be sufficient and will avoid graphical glitches (firing out the wrong hole).